api/v1/openapi.yaml

```
@@ -556,6 +556,18 @@ paths:
           description: Success
           schema:
             "$ref": "#/definitions/SelectorCache"
+  "/lrp":
+    get:
+      summary: Retrieve list of all local redirect policies
+      tags:
+      - service
```

Nit: I think this should be `lrp` or similar instead, this API is doesn't seem related to services.
```suggestion
      - lrp
```

---

I've been considering LRP to be a type of service, you have a virtual IP in front and the traffic to that IP is translated to a "real" IP behind. So `service` seems fine to me.